app/models/piece.rb

```diff
@@ -19,4 +19,19 @@ def move_to!(row, col)
     raise "Out of bounds" if row < 0 || row > 7 || col < 0 || col > 7
     update(row: row, col: col)
   end
+
+  def capture_piece(row, col)
+    other_piece = game.piece_at(row, col)
+    return true if game.path_obstructed? && other_piece.user != user
```

Comment:
The tests are holding me up.  I will put what I have shortly.  For some reason, creating tests is not sticking in my head.